app/views/layouts/_navbar.html.haml

```diff
@@ -1,7 +1,2 @@
-- if user_signed_in?
-  -# = link_to "Hello #{current_user.name}", root_path
-  -# = link_to "Edit", edit_user_registration_path
-  = link_to 'Sign Out',  destroy_user_session_path, method: :delete
-- else
   = link_to 'Sign Up', new_user_registration_path
```

Comment:
I think the conditional should stay. How about this: 
```haml
- if user_signed_in?
   %p= "Logged in as: #[current_user.email}"
- else
  = link_to 'Sign Up', new_user_registration_path
  = link_to 'Login', new_user_session_path
```
Another thought I have is the words, "Sign Up" doeas not match with "Login" IMO . Perhaps you guys should discuss that?